src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -261,9 +261,18 @@ export class WizardModal extends Modal {
 			() => undefined);
 	}
 
-	public open(): void {
+	/**
+	 * Opens the dialog to the first page
+	 * @param source Where the wizard was opened from for telemetry (ex: command palette, context menu)
+	 */
+	public open(source?: string): void {
+		this._telemetryEventService.createActionEvent(TelemetryView.Shell, TelemetryAction.WizardOpen, undefined, source)
```

Comment:
This already calls into modal.show so is already sending a ModalDialogOpened event - you shouldn't need this extra one (the name is what would be used to identify the wizard)